jscomp/build_tests/react_ppx/src/gpr_3987_test.bs.js

```diff
@@ -33,7 +33,7 @@ let Gpr3987ReproOk = {
   make: make
 };
 
-React.createElement(make, makeProps("test", (function (param, param$1) {
+React.createElement(make, makeProps("test", ((param, param$1) => {
```

Comment:
I'm not sure I can remove this parens safely

```suggestion
React.createElement(make, makeProps("test", (param, param$1) => {
```

It would be possible if Lam IR could provide context for the use of IIFE pattern.